qiskit/providers/ibmq/api_v2/rest/schemas/job.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Schemas for job."""
+from marshmallow.validate import OneOf
+from qiskit.providers.ibmq.apiconstants import ApiJobStatus
+from qiskit.validation import BaseSchema
+from qiskit.validation.fields import String, Url, Nested, Number, Boolean
+
+
+# Helper schemas.
+
+class InfoQueueResponseSchema(BaseSchema):
+    """Nested schema for StatusResponseSchema"""
+
+    # Required properties
+    status = String(required=True)
+    position = Number(required=True, missing=0)
+
+
+class JobResponseSchema(BaseSchema):
+    """Nested schema for CallbackUploadResponseSchema"""
+    # pylint: disable=invalid-name
+
+    # Required properties
+    id = String(required=True)
+    kind = String(required=True)
```

Comment:
For `callback_upload` path, I can also see `error`, `creationDate`, and `id` as properties used.

Reply:
Hmmm I could not find anything. The only place I saw is `test_job_submit_object_storage` under `test_account_client_v2` where it only uses: `job['id']` and `job['kind']`.

Reply:
It's line [487-495](https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/qiskit/providers/ibmq/job/ibmqjob.py#L487) in ibmqjob. The `job` in `callback_upload` response is returned as `submit_info` by `job_submit_object_storage`

Reply:
That is so strange, my IDE doesn't show the call in hierarchy!
![image](https://user-images.githubusercontent.com/8620461/63620581-a2aa3580-c5bf-11e9-84b7-406e824cc3ec.png)

However, yes it is called in `_submit_callback` method under `ibmqjob`.

Reply:
Also, I see these two calls:
```
submit_info.get('creationDate')
```
Since it uses `get` method instead of `[]` operator, should I make `creationDate` field optional? Or am I overthinking it?